cytominer_eval/operations/precision_recall.py

```diff
@@ -11,7 +11,9 @@
 
 
 def precision_recall(
-    similarity_melted_df: pd.DataFrame, replicate_groups: List[str], k: int,
+    similarity_melted_df: pd.DataFrame,
```

Comment:
ah, thanks for including these black formatting updates here as well. I must have missed them in my previous contributions